descriptors/ModuleDescriptor-template.json

```diff
@@ -231,6 +231,40 @@
             "orders-storage.sources.item.delete",
             "orders-storage.vendor_details.item.delete"
           ]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/orders/{id}/receiving",
```

Comment:
The path here differs from that in the Orders API sheet, which one is correct?

Reply:
The naming is based on suggestions described in [REST Resource Naming Guide](https://restfulapi.net/resource-naming/):
> Use nouns to represent resources
Taking into account that we have also `receiving-history` endpoint I decided to use `receiving` here...